cygnus-common/spec/SPECS/cygnus-common.spec

```diff
@@ -171,6 +171,10 @@ rm -rf $RPM_BUILD_ROOT
 /var/run/%{_project_name}
 
 %changelog
+* Thu Feb 3 2022 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.16.0
+- [cygnus-commons][NGSIHandler] Check TextUnrestricted type to escape character ' as '' (#2125)
```

Comment:
```suggestion
```

Moving to [cygnus-ngsi] (as in PR #2128 it seems all modifications are under cygnus-ngsi/ directory)